src/app/auth/profiles/profiles.component.ts

```diff
@@ -1,15 +1,9 @@
 import { Component, EventEmitter, OnInit, Output } from '@angular/core';
 import { MatDialog } from '@angular/material';
 import { JsonParseErrorDialogComponent } from './json-parse-error-dialog/json-parse-error-dialog.component';
-import {
-  trigger,
-  state,
-  style,
-  animate,
-  transition
-} from '@angular/animations';
-import { ElectronService } from '../../core/services/electron.service';
+import { trigger, state, style, animate, transition } from '@angular/animations';
```

Comment:
Could you remove the formatting changes which aren't related to this PR?
It obscures the work being done.